project/Build.scala

```diff
@@ -1017,6 +1016,30 @@ object Build {
       }.dependsOn(compile in Compile).evaluated
     )
 
+  lazy val `prepare-community-build` = taskKey[Unit]("Publish local the compiler and the sbt plugin. Also store the versions of the published local artefacts in two files, community-build/{dotty-bootstrapped.version,sbt-dotty.sbt}.")
```

Comment:
call the key prepareCommunityBuild since camel-case is the convention for sbt keys (also no need for keys to be lazy vals).